_docs/services/app-autoscaler.md

```diff
@@ -0,0 +1,472 @@
+---
+parent: services
+layout: docs
+sidenav: true
+title: App-AutoScaler
+name: "app-autoscaler"
+description: "Cloud Foundry App-AutoScaler: scale number of application instances based on metrics"
+status: "Beta"
+---
+
+We are pleased to introduce App-AutoScaler to the platform. [Cloud.gov](https://www.cloud.gov/) has always made it easy to handle increased traffic by scaling your application memory or instances with a single command. With App-Autoscaler, responding to changes in traffic and resource usage is now even easier.
+
+App-AutoScaler adjusts the number of application instances automatically based on metrics like memory used, throughput, and response time. You can also set up a schedule to scale your application instance count at preset times. 
+
+Set a Scaling Policy for the application based on built-in dynamic or custom metrics and App-AutoScaler will scale the number of application instances within the boundary of your organization's memory quota.
+
+
+## Pricing
+
+$5000/annually, memory quotas are still enforced. More information can be found on the [pricing page]({{ site.baseurl }}{% link _pages/pricing.md %}).
+
+## Plans
+
+Service Name | Plan Name | Description | 
+------------ | --------- | ----------- | 
+`app-autoscaler` | `autoscaler-free-plan` | Automatically increase or decrease the number of application instances based on a policy you define. | 
+
+If your organization does not have access to the `app-autoscaler` service in the Cloud Foundry Marketplace (`cf marketplace`), please contact [support@cloud.gov]({{ site.support_email }}) to arrange for a modification to your inter-agency agreement.
```

Comment:
```suggestion
If your organization does not have access to the `app-autoscaler` service in the Cloud Foundry Marketplace (`cf marketplace`), please contact [support@cloud.gov]({{ site.support_email }}) to arrange for a modification to your inter-agency agreement.
```

I think we probably want them to contact inquiries@cloud.gov? There is also a config value for the business email address